packages/instant-apps-components/src/components/instant-apps-app-guide/AppGuide/interfaces/interfaces.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface AppGuidePage {
+  title: string;
+  content: string[];
+  renderContentAs: AppGuideRenderType;
```

Comment:
What do you think of updating the prop name to `type`?

Reply:
Sounds good, updating per our conversation.